### DISCUSSION TITLE
能否提供一种功能（还是这个功能已有？）：挂载某个网盘，Guest 不能访问，或者某个挂载加密？

### DISCUSSION THREAD

**Discussion by ferocknew** [AlistGo/alist#3180] [Ideas]
如题

文档看了很久，可能疏忽了，一直没找到对应的功能实现。
需求场景：

通过挂载某网盘，提供了 webDAV 功能，可以用于各种工具，比如笔记软件 joplin。
但是因为内容较为私有，不希望其他人可以看到。

**Comment by ferocknew** [comment]
元信息，可以变相实现这个功能。
虽然，文件夹最上一层可以看到，但是至少可用，可以 Close 了。
感谢

**Comment by anwen-anyi** [comment]
你是不是在找元信息的隐藏？？？
路径选择root 就是`/` 根目录，把你想隐藏的文件夹隐藏了呗

再新建一个元信息 选择你想加密的文件夹加密 然后勾选上应用到子文件夹

**Comment by ferocknew** [comment]
隐藏，需要写 golang 的正则，这个怎么写？请教
